src/libraries/System.Private.CoreLib/src/System/Reflection/Assembly.cs

```diff
@@ -152,6 +153,7 @@ public virtual IEnumerable<Type> ExportedTypes
         public virtual Assembly GetSatelliteAssembly(CultureInfo culture, Version? version) { throw NotImplemented.ByDesign; }
 
         public virtual FileStream? GetFile(string name) { throw NotImplemented.ByDesign; }
+        [RequiresAssemblyFiles(Message = "'System.Reflection.Assembly.GetFiles(bool)' will throw for assemblies embedded in a single-file app", Url = "https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/il3001")]
```

Comment:
URL pointing to the warning should not be in the `Url` property - VS will do that automatically and everywhere else people can easily search for the warning.
The `Url` property is meant to point to docs specifically about this method - if there's any.